tasks/main.yml

```diff
@@ -6,6 +6,14 @@
   yum:
     name: nfs-utils
     state: present
+  when: ansible_os_family == "RedHat"
+
+- name: Install NFS mount utility
+  become: yes
+  apt:
+    pkg: nfs-common
+    state: latest
```

Comment:
Minor comment. We tend to use `state: present` as above in our roles to enforce idempotent package installation. Not a huge problem here as enabling `molecule` tests for this role in Travis is not easy as mentioned by @joshmoore. Is there a rationale for using `latest` from your side @nickshek ?